### PR TITLE
Telemetry scheduler failed task handling

### PR DESCRIPTION
**Pull request by AmitRoushan**
**What this PR does / why we need it**:
The code handles telemetry failed tasks changes.

Note:
The PR is raised to review the framework changes.
UT need to get added
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by AmitRoushan** [comment]
@NajmudheenCT @sushanthakumar @ThisIsClark 
PTAL

**Review by NajmudheenCT — APPROVED** [review]
LGTM

File: etc/delfin/delfin.conf [hunk 1]
```diff
@@ -13,6 +13,8 @@ db_backend = sqlalchemy
 [TELEMETRY]
 performance_collection_interval = 300
 periodic_job_schedule_interval = 180
+failed_job_schedule_interval = 240
```
Comment (sushanthakumar) [inline etc/delfin/delfin.conf]: These 2 can be moved as constant to TelemetryCollection class

Comment (AmitRoushan) [reply etc/delfin/delfin.conf]: @sushanthakumar 
done as per comment
PTAL

**Comment by sushanthakumar** [comment]
LGTM

**Pull request merged**